### PR TITLE
Updating projects_enhance branch

### PR DESCRIPTION
**Pull request by samus-aran**

### CODE REVIEW

**Comment by haris-raheem** [comment]
looks like my current updates can be auto merged.

File: jssource/src_files/modules/Project/Project.js [hunk 1]
```diff
@@ -99,5 +99,5 @@ function prep_export_to_project(the_form)
 	the_form.sugar_body_only.value='1';
 }
 
- html =  '<a id="create_link" onclick="' + $("#view_gantt").attr('onclick') + '" class="utilsLink">' + $("#view_gantt").attr('value') + '</a>&nbsp;&nbsp;' + $(".moduleTitle .utils").html();
```
Comment (samus-aran) [inline jssource/src_files/modules/Project/Project.js]: Can we just remove these rather than having them commented out?

**Comment by haris-raheem** [comment]
for now, commenting is better because requirements are bouncing back and forth.

**Comment by mattlorimer** [comment]
commented out code will not be accepted, you should use git more effectively.

**Pull request merged**